Applications/MaterialManager/Samples/Create/Boards/Readme.md

```diff
@@ -0,0 +1,27 @@
+<h1 id="createBoardType"> Create board type</h1>
+
+With the Homag Connect materialManager boards client, new board types can be created.
+
+<strong>Example:</strong>
+
+```csharp
+// Create new instance of the materialManager client:
+var client = new MaterialManagerClientMaterialBoards(subscriptionId, authorizationKey);
+
+// Define the new board type request:
+var boardTypeRequest = new MaterialManagerRequestBoardType
+{
+    Code = "BT_White_19mm",
+    Description = "White Board 19mm",
+    Thickness = 19.0,
+    Width = 2800.0,
+    Length = 2070.0,
+    // Add other required properties
```

Comment:
Shouldn't we at least show the required properties in the example?

Reply:
I checked the required properties

Reply:
I meant, because it says // Add other **required** properties 
Found the wording a bit confusing

Reply:
Ah sorry, your changes were not updated in my view